### PR TITLE
[MM-36603] - Text is cut-off on the bottom of the 14-day trial 3 day to end of trial email notification

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR ensures that when cloud trial emails ending warning emails have the Support Email defaults to `feedback-cloud@mattermost.com`

#### Ticket Link
[MM-36603](https://mattermost.atlassian.net/browse/MM-36603)

```release-note
NONE

```

### CODE REVIEW

File: app/email/email.go [hunk 1]
```diff
@@ -591,13 +591,18 @@ func (es *Service) NewEmailTemplateData(locale string) templates.Data {
 		organization = localT("api.templates.email_organization") + *es.config().EmailSettings.FeedbackOrganization
 	}
 
+	supportEmail := *es.config().SupportSettings.SupportEmail
+	if supportEmail == "" && *es.license().Features.Cloud {
+		supportEmail = " feedback-cloud@mattermost.com"
+	}
```
Comment (marianunez) [inline app/email/email.go:597]: We only want to hardcode the support email address to certain cloud emails (the one in the ticket) not all server email notifciations and also regardless of what the support email is configured. Should we do this inside of the template of that email specifically directly instead?

**Review by marianunez — CHANGES_REQUESTED** [review]

File: app/email/email.go (hunk 1 above)
Comment (AGMETEOR) [reply app/email/email.go]: Undid the changes.

**Review by stevemudie — APPROVED** [review]
Looks good!

The text is no longer cut-off on the bottom of the email and it contains the correct email address.

File: app/email/email_test.go [hunk 2]
```diff
@@ -93,3 +94,44 @@ func TestSendInviteEmails(t *testing.T) {
 		verifyMailbox(t)
 	})
 }
+
+func TestSendCloudTrialEndWarningEmail(t *testing.T) {
```
Comment (marianunez) [inline app/email/email_test.go:98]: Great test 👍

**Review by marianunez — APPROVED** [review]
Thanks @AGMETEOR!

**Pull request merged**